fix:Select Widget | Default Value (#681)
appsmithorg/appsmith-docs: website/docs/reference/widgets/select.md

| **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
| **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |

#### Default selected value

> This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  

Reviewer: Please add an example.

> This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  

Author: done

> This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  

Reviewer 2: I think the confusion can be avoided by a small modification in the first line

> This property allows you to specify a default value when the widget is first displayed.

> Please note that this property should be set to match the ```value``` attribute of your desired option from the options property.

Reviewer: Let's not use Please note.
```suggestion
The default selected value should be set to the value of the option from the options property. This value should match the value attribute of the selected option.
```

| **Filterable**            | Makes the dropdown list filterable.                                                                                                                                               |
| **Server Side Filtering** | Enables server-side filtering via an API / Query request. Use this property when the Select widget's Option data is being bound to an API / Query.                                        |

#### Default selected value

This property allows you to specify a default value that can be selected when the widget is first displayed. This can be useful if you want to pre-populate the widget with a specific value, or if you want to ensure that a certain option is selected by default.  

> The default selected value should be set to the value of the option from the options property. This value should match the value attribute of the selected option.

Reviewer 2: ```suggestion
The Default Selected Value should be set to the value of the option from the Options property.
```

these two lines mean the same. Also we can merge this in the first para only.

> For example, if your options property is an array of objects with a **label** and a **value** property, such as

Reviewer 2: Wherever the name of the property is mentioned, keep it title case. eg: here Options property
